acmwebsite/templates/survey_results.xhtml

```diff
@@ -0,0 +1,47 @@
+<html py:strip=""
+  xmlns:py="http://genshi.edgewall.org/"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  py:extends="master.xhtml">
+
+  <head py:block="head" py:strip="True">
+    <title>Mines ACM - Survey ${survey.id} Responses</title>
+  </head>
+
+  <body py:block="body" py:strip="True">
+    <py:import href="acmwebsite.templates.survey_components" alias="comp" />
+
+    <h1 class="page-header" py:content="title" />
+    <h2>Responses</h2>
+    <div class="row">
+      <div class="col-md-12">
+        <table class="table table-striped table-condensed table-bordered">
+          <!--
```

Comment:
Use a removed comment rather than a preserved.

> In Kajiki, you can use normal XML comments, and the comments will exist in the generated markup. If you wish the comments to be removed before rendering the document, you can begin the comment with the syntax `<!–!` instead of `<!–`.

Reply:
That's cool!